pkg/cluster/config.go

```diff
@@ -84,6 +92,20 @@ func (c *Config) DeepCopy() *Config {
 		c2.AWSIAMConfigs[k] = v.DeepCopy()
 	}
 
+	if c.NutanixMachineConfigs != nil {
+		c2.NutanixMachineConfigs = make(map[string]*anywherev1.NutanixMachineConfig, len(c.NutanixMachineConfigs))
+	}
+	for k, v := range c.NutanixMachineConfigs {
+		c2.NutanixMachineConfigs[k] = v.DeepCopy()
+	}
+
+	if c.SnowMachineConfigs != nil {
```

Comment:
Were the Snow adjustments intentional?

Reply:
Yes, they were causing deep equal to fail.